README.md

```diff
@@ -244,48 +102,20 @@ Feel free to check the [issues page](../../issues/).
 
```

Comment:
- [ ] @dicodiaz great work on your readme however am unable to access the readme using the link above, kindly check the URL route to the issues page

Reply:
It's not working because the branch name includes a `/`, which makes it so that it jumps from `https://github.com/dicodiaz/vet-clinic-database/tree/feature/create-animals-table` to `https://github.com/dicodiaz/vet-clinic-database/tree/issues` and the latter is invalid.

This means that it will work when the PR is merged (you can test this in this repo's default branch right [here](https://github.com/dicodiaz/vet-clinic-database#:~:text=%F0%9F%A4%9D-,Contributing,-Contributions%2C%20issues%2C%20and)), so would it make sense to continue the code review? The alternative is to create a new branch and PR, which isn't optimal.

I'll make sure not to include a `/` in my future branch's names. Good catch 😃